homeassistant/components/googlehome/sensor.py

```diff
@@ -0,0 +1,114 @@
+"""
+Support for Google Home alarm sensor.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.googlehome/
+"""
+import logging
+from datetime import timedelta
+
+from homeassistant.components.sensor import ENTITY_ID_FORMAT
+from homeassistant.components.googlehome import (
+    CLIENT, DOMAIN as GOOGLEHOME_DOMAIN, NAME)
+from homeassistant.const import DEVICE_CLASS_TIMESTAMP
+from homeassistant.helpers.entity import Entity, async_generate_entity_id
+import homeassistant.util.dt as dt_util
+
+
+DEPENDENCIES = ['googlehome']
+
+SCAN_INTERVAL = timedelta(seconds=10)
+
+_LOGGER = logging.getLogger(__name__)
+
+ICON = 'mdi:alarm'
+
+SENSOR_TYPES = {
+    'timer': "Timer",
+    'alarm': "Alarm",
+}
+
+
+async def async_setup_platform(hass, config,
+                               async_add_entities, discovery_info=None):
+    """Set up the googlehome sensor platform."""
+    if discovery_info is None:
+        _LOGGER.warning(
+            "To use this you need to configure the 'googlehome' component")
+
+    devices = []
+    for condition in SENSOR_TYPES:
+        device = GoogleHomeAlarm(hass.data[CLIENT], condition,
+                                 discovery_info)
+        devices.append(device)
+
+    async_add_entities(devices, True)
+
+
+class GoogleHomeAlarm(Entity):
+    """Representation of a GoogleHomeAlarm."""
+
+    def __init__(self, client, condition, config):
+        """Initialize the GoogleHomeAlarm sensor."""
+        self._host = config['host']
+        self._client = client
+        self._condition = condition
+        self._name = None
+        self._state = None
+        self._available = True
+
+    async def async_added_to_hass(self):
+        """Subscribe GoogleHome events."""
+        await self.async_init()
+
+    async def async_init(self):
```

Comment:
why isn't this function simply async_added_to_hass ?

Reply:
Yeah! Right :) changing it now.